config/prow/README.md

```diff
@@ -3,10 +3,8 @@
 This directory contains the config for our
 [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) instance.
 
-- `Makefile` Commands to interact with the Prow instance regarding configs and
-  updates. Run `make help` for assistance.
-- `cluster/*.yaml` Deployments of the Prow cluster.
-- `core/*.yaml` Generated core configuration for Prow.
+- `../Makefile` Commands to interact with the Prow instance regarding configs and updates. Run `make help` for assistance.
```

Comment:
It should be cleaner if we just move the `Makefile` to the `prow` folder. Sorry I missed that in my previous change.